src/components/ExpenseAddedAlert.js

```diff
@@ -0,0 +1,27 @@
+const ExpenseAddedAlert = ({ expenseAdded, setExpenseAdded }) => {
+  return (
+    <div
+      role="alert"
+      style={{ zIndex: 1 }}
```

Comment:
do we really need it? if yes, doesn't bootstrap offer any utility classes for this?